### ISSUE TITLE
Add flags for Flutter Analyze Task

### ISSUE THREAD

**Issue by Giuspepe** [hey24sheep/vsts-flutter-tasks#16]
First of all, thank you very much for maintaining this great package! It's helping a lot. :) 

I want to set up a Azure Pipeline which fails if the Flutter analyzer finds any warnings or hints. To do this I could use the Flutter Command task and simply call `flutter analyze --fatal-infos --fatal-warnings`. However, since a Flutter Analyze task already exists, I think It would be cleaner if one could pass these flags (`--[no]-fatal-infos`) etc. directly to the Flutter Analyze task.

**Comment by hey24sheep** [comment]
Hi @Giuspepe,

I have added the `extra-args` instead of a particular flag like `--fatal-infos` or something. Reason being that these flags are only supported on flutter >= 2.0 and many people won't be using it. So, anyone who wants this flag or any other upcoming features will be able to easily include 'n' number of flags. Check the updated readme for usage.

Let me know if there is anything else. Closing in respect to 3d87929
